secretmanager/snippets/create_secret_with_delayed_destroy.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+command line application and sample code for creating a new secret with
+delayed_destroy.
+"""
+
+# [START secretmanager_create_secret_with_delayed_destroy]
```

Comment:
suggestion: move the beginning of the region tag to line 22 (before imports required by the sample), we usually don't show the whole code with execution from shell in the docs.